Adding HTTP Proxy documentation via Config Map (#2401)
kubeflow/website: content/en/docs/other-guides/kubeflow-behind-proxy.md

```shell
kfctl apply -V -f kfctl_k8s_istio.yaml
```

> 
> # To Use a ConfigMap to Configure Kubeflow Proxy
> An alternative to changing the manifests would be to use `PodDefault` to inject the proxy configuration via a (ConfigMap)[https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/] 

Reviewer: Suggestion:

- Fix the link: Markdown's format is `[ a thing ](link)` instead of `(a thing)[link]`
- Use sentence case

```suggestion

# To use a ConfigMap to configure the Kubeflow proxy

An alternative to changing the manifests would be to use `PodDefault` to inject the proxy configuration via a [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/).

```

> A sample config map may look like the following:

Reviewer: ```suggestion
A sample ConfigMap may look like the following:

```
(For consistency)

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: proxies
data:
  HTTP_PROXY: http://my-proxy.exemple.com:3128
  HTTPS_PROXY: http://my-proxy.exemple.com:3128
  NO_PROXY: exemple.com,localhost,cluster.local,192.168.0.0/16,172.16.0.0/12,10.247.0.0/16
```

> Then, create a `PodDefault`:

Reviewer: ```suggestion
Then, create a `PodDefault`:

```

```
apiVersion: kubeflow.org/v1alpha1
kind: PodDefault
metadata:
  name: my-proxies
spec:
  desc: Inject proxies environment variables to get access to Internet
  selector:
    matchLabels:
      my-proxies: "true"
  envFrom:
  - configMapRef:
    name: proxies
```

Now, any pod with the label `my-proxies` will automatically have these environment variables available to them.

> To do so after deployment, you can use `kubectl` 

Reviewer: ```suggestion
To do so after deployment, you can use `kubectl`:
```

```

> # # Assign label 'my-proxies=true' to a single pod
> # kubectl label pods my-pod my-proxies=true     
> 
> # # Assign label 'my-proxies=true' to a group of pods with the label 'group-of-pods'
> # kubectl label pods -l group-of-pods my-proxies=true     

Reviewer: As per @PatrickXYS comment below:

```suggestion
# Assign label 'my-proxies=true' to a single pod
kubectl label pods my-pod my-proxies=true     

# Assign label 'my-proxies=true' to a group of pods with the label 'group-of-pods'
kubectl label pods -l group-of-pods my-proxies=true     
```

> ```

Reviewer 2: I would recommend removing one `#` per code-line

Reviewer: @aronchick Hi, any update on this?

cc @PatrickXYS @RFMVasconcelos
